### ISSUE TITLE
[Feature Request] Collapse/Expand subfolders for Branches

### ISSUE THREAD

**Issue by rarepops** [ForkIssues/TrackerWin#1415]
I would like the ability to collapse all (including children) under a "folder" in the Branches view

**Comment by DanPristupov** [comment]
Like that?
<img width="224" alt="Screenshot 2021-12-16 at 10 40 47" src="https://user-images.githubusercontent.com/618115/146346977-aa48ceac-dd0e-46f2-b888-5d66fd4713a8.png">

Use `/` as a folder separator

**Comment by rarepops** [comment]
Yes, this view. Something like right-click on a root and collapse/expand all (including children)

**Comment by DanPristupov** [comment]
You mean, like that?

<img width="224" alt="Screenshot 2021-12-16 at 10 46 36" src="https://user-images.githubusercontent.com/618115/146347941-17cba4ff-2e4e-4913-94fa-edbfece4ae09.png">

**Comment by rarepops** [comment]
Well, no because that just collapses the root, but not the children as well. 

In the example above, if you had **fixtures** expanded, and had multiple like it, and those had also children, it will take a lot of clicks to collapse them all.

In cases where you have a lot of nesting, I might want to collapse not just the root, because it will navigate better through the roots if the children are collapsed :)

And it's tedious to collapse all children on the current level. So maybe that would be one way. Either collapse all children on the selected root (recursive or not I don't care much)

**Comment by SmithPlatts** [comment]
A context menu item for "expand this and all children" would be very beneficial.

The company I work for as a very ... complicated branch layout, and I'm often having to expand many levels down to get to the actual branch I want to either view (local) or checkout (remote.)
